tests/pages/BasePage.ts

```diff
@@ -8,6 +8,9 @@ export class BasePage {
   }
 
   async navigateTo(url: string) {
-    await this.page.goto(url);
+    await this.page.goto(url, {
+      waitUntil: 'networkidle', // Czeka na moment, w którym brak aktywności sieciowej
+      timeout: 70000            // Timeout na 70 sekund 
```

Comment:
ta linijka nie zadziała, globalnie timeout testu jest ustawiony na 30 sekund, jeśli chcesz zwiekszyć timeout testu dla pojedyńczego testu trzeba to ustawić w w momencie wywoływania testu